source/minini.c

```diff
@@ -49,17 +51,57 @@ static int _minini_handler(void* user, const char* section, const char* name, co
     return 0;
 }
 
+void minini_open_config()
+{
+    // Check to see if there's a config file on the SD Card first
+    minini_config_file = fopen("sdmc:/minute/minute.ini", "r");
```

Comment:
What about stat() to check for files?

Reply:
stat would be an additional call. If it sees the file is there you have to open it anyway. So trying to open and see if it succeeded is simpler and faster.